apps/trading/pages/client-router.tsx

```diff
@@ -196,7 +196,7 @@ export const useRouterConfig = (): RouteObject[] => {
     },
     {
       path: 'portfolio/*',
-      element: <Outlet />,
+      element: <LayoutWithSidebar sidebar={null} />,
```

Comment:
Why this change? I ended up in this state (sidebar open without the option to toggle it off):

![Screenshot 2024-05-14 at 14 58 51](https://github.com/vegaprotocol/frontend-monorepo/assets/6803987/f92c2912-74ab-4820-976f-f9e9fe579c5e)